frontend/www/js/omegaup/components/arena/ContestPractice.vue

```diff
@@ -168,7 +172,23 @@ export default class ArenaContestPractice extends Vue {
   }
 
   onRunSubmitted(run: { code: string; language: string }): void {
-    this.$emit('submit-run', { ...run, problem: this.activeProblem });
+    this.$emit('submit-run', {
+      ...run,
+      problem: this.activeProblem,
+      target: this,
+    });
+  }
+
+  onRunDetails(source: SubmissionRequest): void {
+    this.$emit('show-run', {
+      ...source,
+      request: {
+        ...source.request,
+        hash: `#problems/${
+          this.activeProblemAlias ?? source.request.problemAlias
+        }/show-run:${source.request.guid}/`,
```

Comment:
```suggestion
        }/show-run:${source.request.guid}`,
```

esto _debería_ ir sin el `/` al final.

Reply:
Listo.